src/modules/googleapis/mappers/list-calendar-events.mapper.spec.ts

```diff
@@ -205,8 +205,8 @@ const outputEvent: Event = {
       htmlLink: '',
       created: '2019-04-26T08:53:25.000Z',
       updated: '2019-04-28T18:39:32.321Z',
-      summary: '',
-      description: '',
+      summary: 'extracted',
+      description: 'extracted',
```

Comment:
for the sake of this unit test, fields which are subject of domain extraction return `extracted` by the mocked function, so they are easy to distinguish